cobbler/actions/mkloaders.py

```diff
@@ -67,11 +89,12 @@ def make_ipxe(self):
         """
         Create symlink of the iPXE bootloader in case it is available on the system.
         """
-        if not pathlib.Path("/usr/share/ipxe").exists():
-            self.logger.info("ipxe directory did not exist. Bailing out of iPXE setup!")
+        if not self.ipxe_folder.exists():
+            self.logger.info('ipxe directory did not exist. Please adjust the "bootloaders_ipxe_folder". Bailing out '
+                             'of iPXE setup!')
             return
         symlink(
-            pathlib.Path("/usr/share/ipxe/undionly.kpxe"),
+            self.ipxe_folder.joinpath(pathlib.Path("undionly.kpxe")),
```

Comment:
`pathlib.Path().joinpath()` accepts `str`, no need to create an intermediate `pathlib.Path` instance.
```suggestion
            self.ipxe_folder.joinpath("undionly.kpxe"),
```